tensorboard/plugins/debugger/debugger_plugin_test.py

```diff
@@ -25,13 +25,23 @@
 
 from tensorboard.backend.event_processing import plugin_event_multiplexer as event_multiplexer  # pylint: disable=line-too-long
 from tensorboard.plugins import base_plugin
-from tensorboard.plugins.debugger import debugger_plugin
 from tensorboard.plugins.debugger import debugger_plugin_testlib
 from tensorboard.plugins.debugger import numerics_alert
 
 
 class DebuggerPluginTest(debugger_plugin_testlib.DebuggerPluginTestBase):
 
+  def setUp(self):
+    # Importing the debugger_plugin can sometimes unfortunately produce errors.
+    try:
+      # pylint: disable=g-import-not-at-top
+      from tensorboard.plugins.debugger import debugger_plugin
+      # pylint: enable=g-import-not-at-top
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
Can we use `except ImportError` here? (And throughout.)

Reply:
I was wondering about that - curious about your thoughts ... `ImportError` didn't actually catch the `AttributeError`.

At the same time, from the perspective of a new developer eyeing the code, specifying `AttributeError` might seem arbitrary - what if the module throws a different exception one day?